gimme

```diff
@@ -92,6 +103,21 @@ _sha256sum() {
 	fi
 }
 
+# sort versions, handling 1.10 after 1.9, not before 1.2
+# FreeBSD sort has --version-sort, none of the others do
```

Comment:
Raspbian 9.3 has it. Probably, that's why it does not work on it.

Reply:
(resolved, different issue)